README.md

```diff
@@ -136,7 +136,7 @@ interface of a device, under *Settings -> Device info -> Device ID*.
 #### Shelly 1AddOn configurations
 *Applies to Shelly 1, 1PM*
 * `"sensors"` - 1-3, number of sensors connected to the addon. (default:0 , max: 3)
-* `"type"` - when the relais should act like a thermostat set the type to `"thermostat"` (default: undefined - expose switch & sensors if > 0)
+* `"type"` - when the relais should act like a thermostat set the type to `"thermostat"` (default: undefined - expose switch & sensors if `"sensors"` > 0)
```

Comment:
Great! Maybe we should add another line? With something like: "If you want just thermometers or humidity sensors don't set a type just make sure number of sensors > 0?